site/src/components/Footer/footer.style.js

```diff
@@ -1,235 +1,328 @@
-    text-align: center;
-  }
-  @media (min-width: 481px) and (max-width: 767px) {
-    text-align: center;
-    h3 {
-      font-size: 1.8rem;
-    }
-  }
-`;
+	.footer-sections{
+		flex: auto;
+		padding: 0 1rem;
+		margin-bottom: 1.5rem;
 
-export const Resourceslink = styled.div`
-  display: inline-flex;
-  gap: 0.3rem;
-  flex-direction: column;
-  a {
-    color: #cccccc;
-    font-size: 1rem;
-    width: fit-content;
-    :hover {
-      color: #00b39f;
-    }
-  }
-
-  @media (min-width: 481px) and (max-width: 767px) {
-    align-items: center;
-    a {
-      font-size: 1.1rem;
-    }
-  }
-  @media (max-width: 480px) {
-    align-items: center;
-  }
-`;
+		.section-title{
+			font-size: 1.25rem;
+			margin-bottom: 1rem;
+			color: rgba(255, 255, 255, 0.8);
 
-export const Maildiv = styled.div`
-  display: flex;
-  gap: 2rem;
-  flex-direction: column;
-  justify-content: center;
-  h3 {
-    color: #cccccc;
-  }
-  @media (min-width: 481px) and (max-width: 767px) {
-    text-align: center;
-    padding-top: 2rem;
-  }
-`;
+			.title-link {
+				transition: 0.2s all;
+				color: rgba(255, 255, 255, 0.8);
+				&:hover {
+					color: ${(props) => props.theme.secondaryColor};
+				}
+			}
+		}
 
-export const Copyrightdiv = styled.div`
-  display: flex;
-  width: 65%;
-  border-top: 2px solid white;
-  justify-content: space-between;
-  padding-top: 0.4rem;
-  p {
-    color: rgb(204, 204, 204);
-    font-size: 0.85rem;
-  }
-  @media (max-width: 480px) {
-    flex-direction: column;
-    width: 88%;
-    text-align: center;
-    p {
-      font-size: 0.75rem;
-      margin-bottom: 0;
-    }
-  }
-  @media (min-width: 481px) and (max-width: 767px) {
-    margin-top: 2rem;
-    flex-direction: column;
-    width: 88%;
-    text-align: center;
-    p {
-      font-size: 0.8rem;
-    }
-  }
-  @media (min-width: 768px) and (max-width: 1350px) {
-    font-size: 0.9rem;
-    flex-wrap: wrap;
-    justify-content: center;
-    column-gap: 4rem;
-    margin-bottom: 2rem;
-  }
-`;
+		.section-categories {
+			padding: 0;
+			list-style: none;
+			margin: 0;
+			font-size: .95rem;
 
-export const Icondiv = styled.div`
-  display: flex;
-  gap: 0.5rem;
-  img {
-    padding: 0.1rem;
-    height: 30px;
-    width: 32px;
-    cursor: pointer;
-    filter: grayscale(1) invert(0.25);
-    :hover {
-      filter: grayscale(0) invert(0);
-    }
-  }
-  a {
-    width: fit-content;
-    height: 30px;
-    margin-right: 1rem;
-    cursor: default;
-  }
-  @media (max-width: 767px) {
-    position: relative;
-    bottom: 50rem;
-    img {
-      justify-content: center;
-    }
-    a {
-      margin: auto;
-    }
-  }
-  @media (max-width: 642px) {
-    bottom: 52rem;
-  }
-  @media (max-width: 480px) {
-    bottom: 46rem;
-  }
-`;
+			.category-link {
+				color: #CCCCCC;
+				transition: 0.2s all;
+
+				&:hover{
+					color: ${(props) => props.theme.secondaryColor};
+				}
+			}
+
+			.new-label {
+				background-color: rgba(255, 255, 255, 0.25);
```

Comment:
theme please.

Reply:
Hii @Chadha93 actually this color { background-color: rgba(255, 255, 255, 0.25); } is not defined in theme reference 
so should defined these color in theme ref  and then make a change or we can go with this  background-color: rgba(255, 255, 255, 0.25.